src/content/quickstart/kind/_index.md

```diff
@@ -7,14 +7,24 @@ weight: 100
 ## KIND
 
 [KIND](https://github.com/kubernetes-sigs/kind) is a tool to run local Kubernetes clusters inside Docker container nodes.
+Please note that Docker must be installed and running on your computer.
 
-Submariner provides scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with the Submariner dataplane components deployed on all the clusters.
+Submariner provides (via [Shipyard](https://github.com/submariner-io/shipyard)) scripts that deploy 3 Kubernetes clusters locally - 1 broker and 2 data clusters with the Submariner dataplane components deployed on all the clusters.
```

Comment:
```suggestion
Submariner provides (via [Shipyard](https://github.com/submariner-io/shipyard)) scripts that deploy 3 Kubernetes clusters locally — 1 broker and 2 data clusters with the Submariner dataplane components deployed on all the clusters.
```

Reply:
It's not part of this PR actually, that PR was merged, but I'll be sure to fix it in another one

Reply:
Actually the original is ASCII while yours is UTF-8, why do you think we need this change?

Reply:
The appropriate character here is an em-dash rather than a hyphen, and since we can use UTF-8, we might as well ;-).